include/extractor/profile_properties.hpp

```diff
@@ -12,11 +14,18 @@ const constexpr auto DEFAULT_MAX_SPEED = 180 / 3.6; // 180kmph -> m/s
 
 struct ProfileProperties
 {
+    enum
```

Comment:
`static const` requires a definition some where in cpp file, but i would keep a constant as a value in an unnamed enum in scope of the struct